test_financials.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from unittest.mock import Mock
+from unittest.mock import patch
+import yfinance as yf
+
+class FinancialsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.msft = yf.Ticker("MSFT")
+        self.utils_mock = patch('yfinance.base.utils.get_json', autospec=True).start()
+        self.addCleanup(patch.stopall)
+
+    def tearDown(self):
+        pass
+
+    def test_get_financials_invalid_data_returns_empty_dataframe(self):
+        self.assertTrue(self.msft.financials.empty)
```

Comment:
yf.Ticker("MSFT") get data from web and test if it empty, self.utils_mock seems not be used,